[2023-07-28T16:07:15.196+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monthly_statistics.preprocess_and_load_data manual__2023-07-28T13:38:23.276006+00:00 [queued]>
[2023-07-28T16:07:15.200+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monthly_statistics.preprocess_and_load_data manual__2023-07-28T13:38:23.276006+00:00 [queued]>
[2023-07-28T16:07:15.200+0200] {taskinstance.py:1308} INFO - Starting attempt 2 of 2
[2023-07-28T16:07:15.207+0200] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): preprocess_and_load_data> on 2023-07-28 13:38:23.276006+00:00
[2023-07-28T16:07:15.209+0200] {standard_task_runner.py:57} INFO - Started process 27656 to run task
[2023-07-28T16:07:15.212+0200] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'monthly_statistics', 'preprocess_and_load_data', 'manual__2023-07-28T13:38:23.276006+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/monthly_statistics.py', '--cfg-path', '/var/folders/s3/52f8ptfd1wg8m_7fvtgb9g0w0000gn/T/tmpgftiaiuh']
[2023-07-28T16:07:15.212+0200] {standard_task_runner.py:85} INFO - Job 18: Subtask preprocess_and_load_data
[2023-07-28T16:07:15.259+0200] {task_command.py:410} INFO - Running <TaskInstance: monthly_statistics.preprocess_and_load_data manual__2023-07-28T13:38:23.276006+00:00 [running]> on host gabriel-3.home
[2023-07-28T16:07:15.322+0200] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='monthly_statistics' AIRFLOW_CTX_TASK_ID='preprocess_and_load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-07-28T13:38:23.276006+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-28T13:38:23.276006+00:00'
[2023-07-28T16:08:46.293+0200] {local_task_job_runner.py:291} WARNING - State of this instance has been externally set to None. Terminating instance.
[2023-07-28T16:08:46.298+0200] {process_utils.py:131} INFO - Sending Signals.SIGTERM to group 27656. PIDs of all processes in the group: [27656]
[2023-07-28T16:08:46.299+0200] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 27656
[2023-07-28T16:09:46.299+0200] {process_utils.py:149} WARNING - process psutil.Process(pid=27656, name='Python', status='running', started='16:07:15') did not respond to SIGTERM. Trying SIGKILL
[2023-07-28T16:09:46.300+0200] {process_utils.py:86} INFO - Sending the signal Signals.SIGKILL to group 27656
[2023-07-28T16:09:46.308+0200] {process_utils.py:79} INFO - Process psutil.Process(pid=27656, name='Python', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='16:07:15') (27656) terminated with exit code Negsignal.SIGKILL
[2023-07-28T16:09:46.308+0200] {standard_task_runner.py:172} ERROR - Job 18 was killed before it finished (likely due to running out of memory)
